src/test/cypressjs/cypress/integration/testRuntimeReleaseVersions.js

```diff
@@ -0,0 +1,45 @@
+describe('Test that all runtime release versions are in the correct custom order', () => {
+    before(() => {
+        cy.goToOpenLibertyStarter();
+    });
+
+    it('Test that all runtime release versions are in the correct custom order', () => {
+        let ind = [];
+        let packages = [];
+        let listOfPackLists = [[]];
+        let currInd = 0;
+        let orderGuide = ["Jakarta EE 9",
+        "Web Profile 9",
+        "MicroProfile 5",
+        "Jakarta EE 8",
+        "Java EE 8",
+        "Web Profile 8",
+        "MicroProfile 4",
+        "MicroProfile 3",
+        "Kernel",
+        "All GA Features"]
```

Comment:
```suggestion
        let orderGuide = [
            "Jakarta EE 9",
            "Web Profile 9",
            "MicroProfile 5",
            "Jakarta EE 8",
            "Java EE 8",
            "Web Profile 8",
            "MicroProfile 4",
            "MicroProfile 3",
            "Kernel",
            "All GA Features"]
```

Bring the first index to the rest of the list and 4 space tab the entire list